src/OSPSuite.SimModelSolver_CVODES/SimModelSolver_CVODES.vcxproj

```diff
@@ -95,7 +95,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <ClCompile>
-      <Optimization>Disabled</Optimization>
+      <Optimization>MaxSpeed</Optimization>
```

Comment:
lol

Reply:
I don't think that's why it seems slow. Performance checks are planned in phase 2. Phase 1 nearing completion

Reply:
The change is correct (I didn't mean that **this** was the reason for the slowdon)